test/common_utils.py

```diff
@@ -139,7 +141,8 @@ def assertExpected(self, output, name, prec=None):
                 raise RuntimeError("The output for {}, is larger than 50kb".format(filename))
         else:
             expected = torch.load(expected_file)
-            self.assertEqual(output, expected, prec=prec)
+            rtol = atol = prec or self.precision
```

Comment:
for ref `self.assertEqual` is currently setting `prec` to `self.precision` if it's None (as done above)

and the check was

```
tolerance = prec + prec * abs(a.max())
self.assertLessEqual(max_err, tolerance, message)
```

So the changes LGTM here